.env.example

```diff
@@ -28,6 +28,8 @@ REDIS_LOCK_TTL=1000
 HOME_START_BLOCK=
 FOREIGN_START_BLOCK=
 
+MAX_PROCESSING_TIME=600000
```

Comment:
Is my understanding correct and 10 min is too much for the watchdog timer? Could you estimate max time expected to handle one iteration of watcher?

Reply:
I guess it depends on the number of events processed at the same time. I will try sending several transactions to the kovan-sokol bridge and see how much time is spent processing them.

Reply:
Sent 10 transactions to Sokol. 2 of them were processed in one block, 8 in another. The time difference between the first log, and the log of the sender receiving the last transactions to send was 3 seconds (that is Sokol's block time, isn't it?).

The time is going to depend on the chain, but I guess one minute should be enough in the vast majority of cases (and it can be configured, anyway).

Reply:
Hmm. I think the timout should be comparable with the polling time. I think 3 or 4 times more than the polling time of the corresponding network. What do you think?

Reply:
Yes, that sounds reasonable. You mean using that as the example, or that the timeout should be computed from the polling time?

Reply:
Yes, let's set the timeout in the example to be 4 times more that the longest polling time and provide these recommendations in README.

Reply:
Please check 3171c1e6b80ece4c289d677d4081c1da213caa27.